Latex/Insight.bib

```diff
@@ -1932,7 +1932,7 @@ @InProceedings{Jolly2003
 
 @Book{Kaandorp1994,
   Author         = "J.A. Kaandorp",
-  Title          = "Fractal Modelling. Growth and From in Bilogy",
+  Title          = "Fractal Modelling. Growth and Fxprom in Biology",
```

Comment:
Fxprom: Form.

Reply:
Oops, editor was in insert mode and not in command mode.